submissions/EmmanuelUdoh/phpTask/fetch.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require "config.php";
+/**
+ * This file Handles Php Data Object connection to Db
+ * which can be of any type e.g SQL db and runs commands in init.sl
+ */
+
+try {
+    session_start();
+    $connection = new PDO("mysql:host=$host", $username, $password, $options);
+    $stmt = $connection->query("SELECT * FROM kimaj.brands");
+    $brands = array();
+    while ($item = $stmt->fetch()) {
+        array_push($brands, $item['brand_name']);
+    }
+    $_SESSION['brands'] = $brands;
```

Comment:
You do not have to use this fetch.php file and the Session Object. You can move your query for brands to the file where you need it. i.e your create.php file